src/Refactoring-Core/RBPullUpMethodRefactoring.class.st

```diff
@@ -96,10 +96,11 @@ RBPullUpMethodRefactoring >> checkInstVarsFor: aSelector [
 	class instanceVariableNames do: 
 			[:each | 
 			((class whichSelectorsReferToInstanceVariable: each) includes: aSelector) ifTrue: 
-					[self
-						refactoringError: ('<1p> refers to <2s> which is defined in <3p>' expandMacrosWith: aSelector
+					[ (self confirm: ('<1p> refers to #<2s> which is defined in <3p>. Do you want push up variable #<2s> also?' expandMacrosWith: aSelector
 								with: each
-								with: class)]]
+								with: class)) 
+						ifTrue: [ self pushUpVariable: each ]
+						ifFalse: [ self refactoringError: 'Do you want push up anyway?' ] ]]
```

Comment:
You could use the following text: 

'You are about to push your method without the instance variable it uses. It will bring the system is an inconsistent state. But this may be what you want. 
So do you want to push up anyway?'